doc/stanzas/executable.rst

```diff
@@ -286,3 +287,32 @@ well:
   executable. The list of names must be of the same length as the list in the
   ``(names ...)`` field. Moreover, you can use ``-`` for executables that
   shouldn't be installed.
+
+However, using ``executables`` the executables defined in the stanza are
+allowed to share modules.
+
+Given modules ``Foo``, ``Bar`` and ``Baz`` the usage of ``executables`` can
+simplify the code:
+
+.. code:: dune
+
+          (executables
+            (names foo bar))
```

Comment:
the indentation seems a bit large here. Shouldn't it be aligned with `code::`?

Reply:
I copied the indentation level from the code snippet in "Linking modes", but I agree it is a bit weird. What is the correct indentation? Then I can fix all occurrences in this file.

Reply:
It should line up with`code` which is 3 spaces indented. We could also set the default syntax highlighting for the entire file to be `dune` then we wouldn't need to indent by 3 but by 2.

Reply:
@emillon knows how to set the default indentation, I couldn't remember where I saw it though.

Reply:
Ok, I've adjusted all the blocks to align with `code`.

Reply:
yes, in that case for now we align to 3. We can't set a default highlighting here since this file for now is included from another one.